apps/molgenis-components/src/components/utils.ts

```diff
@@ -134,3 +134,34 @@ export function convertToPascalCase(string: string): string {
   });
   return result;
 }
+
+export function getLocalizedLabel(
+  tableOrColumnMetadata: ITableMetaData | IColumn,
+  locale: string
+) {
+  let label;
+  if (tableOrColumnMetadata?.labels) {
+    label = tableOrColumnMetadata.labels.find(
+      (el) => el.locale === locale
+    )?.value;
+    if (!label) {
+      label = tableOrColumnMetadata.labels.find(
+        (el) => el.locale === "en"
+      )?.value;
+    }
+  }
+  if (!label) {
+    label = tableOrColumnMetadata.name;
+  }
+  return label;
+}
+
+export function getLocalizedDescription(
```

Comment:
It's good practice to add the return types to the function e.g. function getLabel(...): string {....}